ui-admin/src/study/surveys/FormHistoryModal.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useId, useState } from 'react'
+import Modal from 'react-bootstrap/Modal'
+import InfoPopup from 'components/forms/InfoPopup'
+import LoadingSpinner from 'util/LoadingSpinner'
+import Select from 'react-select'
+import { instantToDefaultString, VersionedForm } from '@juniper/ui-core'
+import { StudyEnvContextT, studyEnvFormsPath } from '../StudyEnvironmentRouter'
+import { useLoadingEffect } from 'api/api-utils'
+import Api from 'api/api'
+import { Button } from 'components/forms/Button'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faArrowRightFromBracket } from '@fortawesome/free-solid-svg-icons'
+
+/**
+ *
```

Comment:
empty jsdoc

Reply:
I've been wondering if we still want to require JSDocs. Do you all find them useful?

Reply:
I'm torn -- I generally find them useless when I'm looking directly at the code, but they're really useful for giving hover tool tips when you're hovering over the function name when it's called from somewhere else.